plugins/ccCommandLineInterface.h

```diff
@@ -207,11 +226,10 @@ class ccCommandLineInterface
 	//! Exports a cloud or a mesh
 	/** \return error string (if any)
 	**/
-	virtual QString exportEntity(	CLEntityDesc& entityDesc,
-									QString suffix = QString(),
-									QString* outputFilename = nullptr,
-									bool forceIsCloud = false,
-									bool forceNoTimestamp = false) = 0;
+	virtual QString exportEntity(CLEntityDesc& entityDesc,
+		const QString &suffix = QString(),
+		QString* outputFilename = nullptr,
+		ccCommandLineInterface::ExportOptions options = ExportOption::NoOptions) = 0;
```

Comment:
Does this interface change require changes elsewhere? Any plugins?

(Honestly I've never looked at the command line stuff, so I don't know what it impacts.)

Reply:
It required changing in a few places, like in the qM3C2 plugin I believe I got them all, but they won't compile if I missed them. so Travis should show it